src/app/app.component.ts

```diff
@@ -19,6 +19,7 @@ export class AppComponent implements OnInit {
   disableRangeSelection = false;
   isDesktop = false;
   selectWithShortcut = false;
+  dragOverItems = false;
```

Comment:
The default should be `true`, otherwise this is a breaking change and requires a new major release, while `true` allows us to publish a new minor version.